### PR TITLE
Improve handling of singular LD matrices

### PR DESCRIPTION
**Pull request by zmccaw-insitro**
Found empirically that the previous regularization (`eps = 1e-4`) applied to singular LD matrices was insufficient. Increasing to `eps = 1` resolves the issue, even in the extreme case that an entire row and column of the LD matrix is 0.

### CODE REVIEW

File: src/sumstats.cpp [hunk 1]
```diff
@@ -91,7 +91,7 @@ SEXP IVWCpp(
     arma::colvec se_anno = se.elem(key);
     arma::mat d_anno = arma::diagmat(se_anno);
     arma::mat v_anno = d_anno * ld_anno * d_anno;
-    arma::mat v_anno_inv = arma::inv_sympd(v_anno);
+    arma::mat v_anno_inv = arma::pinv(v_anno);
```
Comment (ctk3b) [inline src/sumstats.cpp:94]: Is this change strictly related to the eps adjustment? Could you comment on the motivation for swapping this out?

**Review by ctk3b — COMMENTED** [review]
Defer to @tsoare-insitro for final review but this seems reasonable. Left one question about a potentially unrelated change

File: src/sumstats.cpp (hunk 1 above)
Comment (zmccaw-insitro) [reply src/sumstats.cpp:94]: This ensures the function behaves as expected if `eps` were set to 0.

Comment (ctk3b) [reply src/sumstats.cpp:94]: Ah makes sense. Worth leaving a comment?

Comment (zmccaw-insitro) [reply src/sumstats.cpp:96]: Sure, just added.

**Review by tsoare-insitro — APPROVED** [review]
LGTM

**Comment by zmccaw-insitro** [comment]
Thanks both!
